agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/TransgenicAlleleIndexer.java

```diff
@@ -78,7 +76,7 @@ private void addGenesToMap(List<Gene> genes, Map<Gene, Set<GeneTransgenicAlleleS
 
 	private List<TransgenicAlleleSummaryDocument> indexTransgenicAlleleSummary() {
 		SearchResponse<TransgenicAlleleDTO> searchResponse = transgenicAlleleApi.findDocuments(0, 0, params);
-		ProcessDisplayHelper display = new ProcessDisplayHelper(2000);
+		ProcessDisplayHelper display = new ProcessDisplayHelper(20000);
```

Comment:
@cmpich if you just take this out it will go to the default of 30seconds

Reply:
Sounds good.